hrm-domain/hrm-core/autoPaginate.ts

```diff
@@ -29,22 +29,22 @@ type LimitAndOffset = {
   offset: number;
 };
 
-/**
```

Comment:
I would prefer us to be moving shared utility code like this away from hrm-core into /packages modules over time, rather than moving more of it into hrm-core. 

hrm-core is its current form an interim compromise, one of those horrible 'amorphous code blob' library modules with loads of things in it because breaking it up would have been too much refactoring to do in addition to moving it out of hrm-service. But we should be aiming to only be putting code specific to individual endpoint implementations in there going forward and moving shared utility code out to packages